text/translation-review-and-correction.liquid.html

```diff
@@ -5,46 +5,52 @@
     border: 1px solid black;
     border-collapse: collapse;
     padding: 5px;
+    min-width: 100%;
+    background-color: white;
   }
 </style>
 
 <crowd-form>
+  <full-instructions header="Translation Quality Check Instructions">
+    <p>The source document has been broken up into sentences and each sentence has been translated using <a href="https://aws.amazon.com/translate/" target="blank" >Amazon Translate</a>. The source text, along with their
+      <strong>machine generated translation</strong> is shown in the table. Please review the translated text and make changes so the read more like something a person would write rather than an automated translation.
```

Comment:
typo: "so the read more" should be something like "so it reads more"

Reply:
Done and added a Strong tag around Submit to emphasize that it's a button.